index.html

```diff
@@ -1325,6 +1327,45 @@ <h2>
 detail how revocation is performed and tracked.
       </p>
 
+      <p>
+All public key formats MUST be expressed in either JSON Web Key (JWK)
+format using the <code>publicKeyJwk</code> property or one of the native
+formats listed in the table below. All other key formats MUST NOT be used.
+      </p>
+
+      <p class="issue">
+The Working Group is still debating whether the default base encoding format
+will be Base58 (Bitcoin) or base64url. The entries in the table below
+currently assume Base58 (Bitcoin), but may change to base64url once the
+group achieves consensus on this particular issue.
+      </p>
+
+      <table class="simple">
+        <thead>
+          <tr>
+            <th>
+Key&nbsp;Type
+            </th>
+            <th>
+Support
+            </th>
+          </tr>
+        </thead>
+
+        <tbody>
+          <tr>
+            <td>
+RSA
+            </td>
+            <td>
+RSA public key values MUST be encoded in Privacy Enhanced Mail
```

Comment:
```
RSA public key values MUST be encoded in Privacy Enhanced Mail
(PEM) format using the <code>publicKeyPem</code> property if they are not
encoded in JWK format.
```
->
```
RSA public key values MUST be encoded in Privacy Enhanced Mail
(PEM) format using the <code>publicKeyPem</code> property or in
JSON Web Key (JWK) format using the <code>publicKeyJwk</code>
property.
```

Reply:
While I agree with your wording, the consensus position of the group on @selfissued's proposal was to not word things in the way that you worded them. What you are suggesting is more or less what we had before, and the group straw polled to change it so that we don't have normative statements about JWK's usage in the table.

Reply:
I do not recall such a straw poll (which [you refer to as a resolution](https://github.com/w3c/did-core/pull/100#discussion_r351089798) above, where you cite the special Monday call, which I was not able to join). I do not think I would have agreed with it, given the result, and I do not recall it being mentioned in the regular Tuesday call. Can you provide a specific pointer to the relevant section of minutes?

In any case -- I'm suggesting the text be changed because the current wording is clumsy at best, and confusing at worst.  Either the table should *only* mention the alternative encodings (leaving all mention of JWK in the preceding text), *or* it should be fully explicit (and mention both/all encoding options for each public key in the row), as in the table text I've suggested.

Reply:
Applied wording that you seemed to be ok with in another thread to this @TallTed.